src/external/fees/FeeManager_v1.sol

```diff
@@ -265,30 +266,90 @@ contract FeeManager_v1 is
     function setDefaultCollateralFee(uint _defaultCollateralFee)
         external
         onlyOwner
-        validFee(_defaultCollateralFee)
     {
-        defaultCollateralFee = _defaultCollateralFee;
-        emit DefaultCollateralFeeSet(_defaultCollateralFee);
+        _setDefaultCollateralFee(_defaultCollateralFee);
     }
 
     /// @inheritdoc IFeeManager_v1
     function setDefaultIssuanceFee(uint _defaultIssuanceFee)
         external
         onlyOwner
+    {
+        _setDefaultIssuanceFee(_defaultIssuanceFee);
+    }
+
+    /// @inheritdoc IFeeManager_v1
+    function setCollateralWorkflowFee(
```

Comment:
Why exactly are you adding an internal function here?

Reply:
You're right, this one is redundant, because it is not used in the `init`. I'll remove it